packages/admin-next/dashboard/src/components/layout/notifications/notifications.tsx

```diff
@@ -31,10 +57,93 @@ export const Notifications = () => {
       </Drawer.Trigger>
       <Drawer.Content>
         <Drawer.Header>
-          <Heading>Notifications</Heading>
+          <Heading>{t("notifications.domain")}</Heading>
         </Drawer.Header>
-        <Drawer.Body>Notifications will go here</Drawer.Body>
+        <Drawer.Body className="overflow-y-auto">
+          {notifications?.map((notification) => {
+            return (
+              <Notification key={notification.id} notification={notification} />
+            )
+          })}
+        </Drawer.Body>
       </Drawer.Content>
     </Drawer>
   )
 }
+
+const Notification = ({
+  notification,
+}: {
+  notification: HttpTypes.AdminNotification
+}) => {
+  const data = notification.data as unknown as NotificationData | undefined
+
+  // We need at least the title to render a notification in the feed
+  if (!data?.title) {
+    return null
+  }
+
+  return (
+    <>
+      <div className="align-center mt-2 flex flex-row justify-center">
+        <div className="text-ui-fg-muted mr-3">
+          <InformationCircleSolid />
+        </div>
+        <div className="w-full">
+          {/* We set the line height to 1 so the icon and text are aligned */}
+          <div className="align-center mb-1 flex flex-row justify-between">
+            <Label weight="plus" className="leading-4">
+              {data.title}
+            </Label>
+            <Text
+              as={"span"}
+              className="text-ui-fg-subtle leading-4"
+              size="small"
+            >
+              {formatDistance(notification.created_at, new Date(), {
+                addSuffix: true,
+              })}
+            </Text>
+          </div>
+
+          {!!data.description && (
+            <Text
+              className="text-ui-fg-subtle whitespace-pre-line"
+              size="small"
+            >
+              {data.description}
+            </Text>
+          )}
+          <NotificationFile file={data.file} />
+        </div>
+      </div>
+
+      <Divider className="my-4" />
```

Comment:
@kasperkristensen since the drawer body has padding, the Divider is not full width (I can force it into being so, but I was wondering what's the pattern we've followed in other places)

Reply:
Answered this another comment, but all classNames can be overwritten

Reply:
Ok cool, I wasn't sure if we want to do such overrides, but if we're good with it I'll do it 👍

Reply:
Thats the intended way to use `@medusajs/ui`. The components implement the common design, but we merge classNames, so when needed you can alter it to fit your use case 👍

Reply:
should we make dividers full width as per design?

Reply:
@fPolic they are now, sorry i didn't update the screenshot

Reply:
> should we make dividers full width as per design?
yes :D